src/settings.php

```diff
@@ -17,7 +17,7 @@
         ],
 
         'db' => [
-            'host' => 'mysql:host=db;',
+            'host' => 'mysql:host=127.0.0.1;',
```

Comment:
Leave these settings unchanged in master, you can just add a new commit that changes it back to db. The live site uses the config in master so it needs to remain unchanged.

Reply:
Changed host back to db, ready for master implementation.